src/lib/turtle.js

```diff
@@ -1112,6 +1129,14 @@ function generateTurtleModule(_target) {
 
         proto.$onclick = function(method, btn, add) {
             this.getManager("mousedown").addHandler(method, add);
+            // Set a notification flag directly to the Sk.TurtleGraphics object if listening
+            // or send a notification event when listening stops
+            if(method != undefined) {
```

Comment:
Is it possible for method to be undefined here?  The line just above looks like it uses it to add a handler, so will that still be valid anyway if method is undefined?  I'm not sure exactly what addHandler is doing.

Reply:
Yes. So in Python, the "onclick" listener can be cancelled out by calling `onclick(None,...)` which transfers to JS as having an undefined function.

Reply:
Note: the addHandler will destroy existing handlers so if the function is undefined (in JS, then), it won't be added to the event listener for mousedown, and therefore no function will be registered anymore to the listener. Here is the code extract:

![image](https://github.com/PwtKCL/skulpt/assets/74306536/b87b1c08-230c-4f67-a743-851359fe567f)